profiler/src/main/java/com/splunk/opentelemetry/profiler/Configuration.java

```diff
@@ -20,74 +20,71 @@
 import static com.splunk.opentelemetry.SplunkConfiguration.PROFILER_MEMORY_ENABLED_PROPERTY;
 import static java.util.logging.Level.WARNING;
 
-import com.google.auto.service.AutoService;
-import io.opentelemetry.sdk.autoconfigure.spi.AutoConfigurationCustomizer;
-import io.opentelemetry.sdk.autoconfigure.spi.AutoConfigurationCustomizerProvider;
+import com.splunk.opentelemetry.SplunkConfiguration;
 import io.opentelemetry.sdk.autoconfigure.spi.ConfigProperties;
 import java.time.Duration;
-import java.util.HashMap;
-import java.util.Map;
+import java.util.function.Function;
 import java.util.logging.Logger;
 
-@AutoService(AutoConfigurationCustomizerProvider.class)
-public class Configuration implements AutoConfigurationCustomizerProvider {
+public class Configuration {
   private static final Logger logger = Logger.getLogger(Configuration.class.getName());
-  private static final boolean HAS_OBJECT_ALLOCATION_SAMPLE_EVENT = getJavaVersion() >= 16;
 
-  private static final String DEFAULT_RECORDING_DURATION = "20s";
-  public static final boolean DEFAULT_MEMORY_ENABLED = false;
-  public static final Duration DEFAULT_CALL_STACK_INTERVAL = Duration.ofSeconds(10);
-  public static final boolean DEFAULT_INCLUDE_INTERNAL_STACKS = false;
-  public static final boolean DEFAULT_TRACING_STACKS_ONLY = false;
-  private static final int DEFAULT_STACK_DEPTH = 1024;
-  private static final boolean DEFAULT_MEMORY_EVENT_RATE_LIMIT_ENABLED = true;
```

Comment:
When someone asks what the default value behavior is for `foo`, you're going to have to go hunting now. I think this makes it way less findable, but whatever.